nix-installer.nix

```diff
@@ -0,0 +1,172 @@
+{ installationSlug ? "nix-install", installedPackages, postInstall ? null }:
```

Comment:
(nitpick): these files should go somewhere inside `installers` like `installer/nix`